packages/augur-ui/src/modules/app/actions/init-augur.ts

```diff
@@ -38,56 +47,75 @@ function pollForAccount(
   getState: () => AppState
 ) {
   const windowApp = windowRef as WindowApp;
-  const { loginAccount } = getState();
-  let accountType =
-    loginAccount && loginAccount.meta && loginAccount.meta.accountType;
-  let usingMetaMask = accountType === ACCOUNT_TYPES.WEB3WALLET;
-  if (!accountType && isGlobalWeb3()) {
-    usingMetaMask = true;
-  }
-  setInterval(() => {
-    const { authStatus, connection } = getState();
-    if (connection.isConnected) {
-      let loggedInAccount: string = undefined;
-      let loggedInAccountType: string = undefined;
-      if (!loginAccount.address) {
-        if (windowApp.localStorage && windowApp.localStorage.getItem) {
-          loggedInAccount = windowApp.localStorage.getItem('loggedInAccount');
-          loggedInAccountType = windowApp.localStorage.getItem('loggedInAccountType');
+
+  let attemptedLogin = false;
+  let intervalId = null;
+
+  async function attempLogin() {
+    const { connection } = getState();
+    if (attemptedLogin) {
```

Comment:
Minor thing, should this only process if UI isConnected?
It would make the code a little more readable

Reply:
good call didn't notice that